src/main/java/org/myrobotlab/service/data/Locale.java

```diff
@@ -162,10 +166,16 @@ public String getDisplayCountry() {
 
   public String toString() {
     return getTag();
+  } 
+
+  public static Properties loadLocalizations(String fullPath) {
+    Properties props = new Properties();
+    try {
+      props.load(new InputStreamReader(new FileInputStream(fullPath), Charset.forName("UTF-8")));
+    } catch(Exception e) {
+      /* don't care common use case */
```

Comment:
There should at a minimum be a log message here with the exception.

Reply:
I'll update it to an a debug, but its not an error or even a warning

Reply:
Why woulnd't it be an error message?  if the file system croaks on you.. this would throw an IO exception.A log message should be here.  Also.. what is closing the Input stream readers, aren't you leaking file handles here?